runtime/lua/vim/_defaults.lua

```diff
@@ -200,6 +200,171 @@ do
       desc = 'Open a floating window showing diagnostics under the cursor',
     })
   end
+
+  --- vim-unimpaired style mappings. See: https://github.com/tpope/vim-unimpaired
+  do
+    -- Quickfix mappings
+    vim.keymap.set('n', '[q', function()
+      vim.cmd.cprevious({ count = vim.v.count1 })
+    end, {
+      desc = ':cprevious',
```

Comment:
(not a blocker) most of these mappings would probably benefit from regular `vim.cmd[[nnoremap ...]]` definitions instead of Lua functions. The `desc` for these Lua mappings is a workaround, but actually gives less clarity for no advantage.

Reply:
I disagree. We need to leave that esoteric DSL in the dust.

Reply:
> can't we pass a string instead of a function for RHS? That would clean this up a lot

forgot about that!

Reply:
>If we want to avoid the desc, then can't we pass a string instead of a function for RHS? That would clean this up a lot and make each mapping one line.

Some of these are nontrivial on a single line and would still require a description. I'm not sure why we're so concerned about avoiding the description or keeping things to one line. Using `vim.cmd` is structured, explicit with counts (avoiding the `:` vs `<Cmd>` distinction), and lets us keep the implementation and human readable description separate.